hibernate-reactive-core/src/main/java/org/hibernate/reactive/session/impl/ReactiveStatelessSessionImpl.java

```diff
@@ -998,10 +998,10 @@ public <R> ReactiveMutationQuery<R> createReactiveMutationQuery(CriteriaDelete<R
 	}
 
 	@Override
-	public <R> ReactiveMutationQuery<R> createReactiveMutationQuery(JpaCriteriaInsertSelect<R> insertSelect) {
+	public <R> ReactiveMutationQuery<R> createReactiveMutationQuery(JpaCriteriaInsert<R> insert) {
 		checkOpen();
-		try {
```

Comment:
Thanks! The PR looks good, but could you add the blank space here before  I merge it?